e2e_playwright/st_tabs_test.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
```

Comment:
super-nit: does the extra line come from ruff?

Reply:
Idk.. I'll try deleting it, and see what happens :D